youtube_dl/extractor/infomaniak.py

```diff
@@ -0,0 +1,88 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    parse_duration,
+)
+
+
+class InfomaniakVod2IE(InfoExtractor):
+    IE_NAME = 'infomaniak:vod2'
+    _VALID_URL = r'https?://player\.vod2\.infomaniak\.com/embed/(?P<id>[0-9a-z]+)'
+    _TEST = {
+        'url': 'https://player.vod2.infomaniak.com/embed/1jhvl2uqg6ywp',
+        'info_dict': {
+            'id': '1jhvl2uqg6xis',
+            'display_id': '1jhvl2uqg6ywp',
+            'ext': 'mp4',
+            'title': 'Conférence à Dyo, octobre 2022',
+            'thumbnail': r're:^https?://.*\.(?:jpe?g|png)$',
+        },
+        'params': {
+            'skip_download': True,
+        },
```

Comment:
There's a non-adaptive format, so we can have file download check (the test suite just gets an initial chunk, so the test won't be fetching the whole 4GB). If the `best`/`http` format isn't automatically the best (but it should be, I suggest: see below), you could put `'format': 'http',` in `params`.

```suggestion
        'md5': '08c3a89906b70a614fa0fdb057e8a22e',
        'info_dict': {
            'id': '1jhvl2uqg6xis',
            'display_id': '1jhvl2uqg6ywp',
            'ext': 'mp4',
            'title': 'Conférence à Dyo, octobre 2022',
            'thumbnail': r're:^https?://.*\.(?:jpe?g|png)$',
        },
```

Reply:
Good point. I added an md5 to _TEST and switched the test params to prefer the direct file ('format': 'http') so the test suite can do the lightweight chunk download without pulling the full video.